CenterForOpenScience/osf.io
# [feature] Close ERPC [OSF-7843]

#### Purpose
- Close ERPC by preventing new draft registrations from being created with the ERPC metaschema.

#### Changes
- Add `enabled` boolean field to `MetaSchema` model and serializer.
- Update `ensure_schemas()` to use Django's `update_or_create()`.
- Prevent draft registrations from being created with disabled metaschemas (in APIv1 and APIv2).

#### Side Effects / Notes
- Existing ERPC draft registrations will still be able to be registered. See ticket for discussion.
- The `/v2/metaschemas/` endpoint will return only active metaschemas that are of the latest version.
- The `/v2/metaschemas/<schema_id>/` endpoint will return any metaschema, including previous versions and inactive schemas.

#### QA Notes
- Confirm ERPC draft registrations cannot be created on the OSF or through the API.
- Confirm ERPC is not returned from `/v2/metaschemas/`
- Confirm 200 (not 404) when requesting ERPC detail (`/v2/metaschemas/<erpc_schema_id>/`)

#### Ticket
- [OSF-7843](https://openscience.atlassian.net/browse/OSF-7843)
